packages/db/src/resources/networks.ts

```diff
@@ -34,5 +47,79 @@ export const networks: Definition<"networks"> = {
       height: Int!
       hash: String!
     }
-  `
+  `,
+  resolvers: {
+    Network: {
+      possibleAncestors: {
+        resolve: async ({id}, {limit = 5, alreadyTried}, {workspace}) => {
+          const network = await workspace.get("networks", id);
+          const result = await workspace.find("networks", {
+            selector: {
+              "historicBlock.height": {
+                $gte: null,
+                $lt: network.historicBlock.height,
+                $ne: network.historicBlock.height
```

Comment:
Yes it should imply that! For some reason I was unable to get this to work otherwise. Happy to revisit it as a separate task but didn't want to spend more time on it as part of this PR.